1.8/s2i/bin/assemble

```diff
@@ -9,9 +9,17 @@ if [ -d ./nginx-cfg ]; then
   echo "---> Copying nginx configuration files..."
   if [ "$(ls -A ./nginx-cfg/*.conf)" ]; then
     cp -v ./nginx-cfg/*.conf "${NGINX_CONFIGURATION_PATH}"
+    chmod -Rf 777 ${NGINX_CONFIGURATION_PATH}
```

Comment:
In order to change configuration in running container we would need to have write access for default user. Files are created as root, so we can:
- Change file owner to default (risky as container can use different user)
- Make files writable by others by setting `666` 👿  mask on file

Making files writable do not impose security risk. If attacker would get access to the image he can switch configuration to any configuration file he wants and then send HUP signal to the nginx process to reload it or use process swapping techniques to change nginx process.